README.md

```diff
@@ -26,3 +26,134 @@ a single-page app that goes through the major features of `git`
 an on-line Lynda.com course; it's free if you first authenticate
 with your U of M credentials via http://lynda.umn.edu
 
+The discussion below assumes that people are paired up in the lab, and
+there will be times where one person will need to log out and let their
+partner login so that everyone has a chance to go through all the
+activities.
+
+:bangbang: This repository will be private so only people in the class
+(and the faculty and TAs) will have access to this contact info.
+Everyone in the class will have access to it, though, so you should
+put some thought in what you want to share.
+
+# Create a contact info page for (each of) you
+
+This repository has a `contact_info` directory, and in this part of the
+lab we'll create a new file in that directory for every person in the
+class, containing their personal contact info.
+
+:question: KK – do we want to have person X type up person Y's info
```

Comment:
I think your idea is a good one. It will also cause people to have to talk and listen... both useful skills for communication.